### DISCUSSION TITLE
Change NODE_ENV in production

### DISCUSSION THREAD

**Discussion by antonykamp** [blitz-js/blitz#2263] [Q&A] [answered]
Hey,

I have a small question concerning the `NODE_ENV` variable :) 

At the moment, it depends on `NODE_ENV` whether the application sends an email or creates a preview of the message (like in the blitz default project). Let's assume that a "main"- and "develop"-environment exists (both run with `blitz build`, `blitz start`). 
What I want to do is sending emails with the main- and creating previews with the develop-environment :/ I tried to set `NODE_ENV` manually, but it looks like blitz (or something else) overwrites this to `production`. 

Do you guys know a nice workaround? :) Maybe the mailer should depend on something else 🤷 
Thank you!

**Comment by flybayer** [comment, marked as answer]
We inherit that behavior from next.js. Here's what I do:

```
# .env
APP_ENV=development

# .env.production
APP_ENV=production

# .env.test
APP_ENV=test
```

And then you can set custom `APP_ENV` values like `STAGING`

**Reply by antonykamp** [reply]
That's exactly what I was looking for! 🥳 
Thank you!
